docs/Makefile

```diff
@@ -20,3 +20,16 @@ html: Makefile
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+# Gen man pages
+ZFS_GIT_REPO = https://github.com/openzfs/zfs.git
+ZFS_GIT_DIR = ./_build/zfs
```

Comment:
Always generating the man page from master is the right first step.  But I think we should consider early on how we're going to handle generating documentation for multiple versions of OpenZFS.  Adding a "Docs by version" section to the sidebar as was done for the [Python documentation](https://docs.python.org/3/) would be a nice way to handle this.  Let's make sure to leave ourselves some options since I'm not sure how difficult it would be to add this kind of thing latter.

Reply:
You're right, I thought about it too. Looks like it would be pretty easy, because now we generate docs. We may use one of sphinx modules for that, for ex. https://github.com/sphinx-contrib/sphinxcontrib-versioning to get versioned docs + man pages via script addtions.

Worst thing that may get us is different links and the need to manage redirects. In this case I prefer some 404 links from other sites but we get faster publication -> better docs now.